### ISSUE TITLE
Remote-SSH not showing ssh logs even when .ssh/config LogLevel is DEBUG2

### ISSUE THREAD

**Issue by studgeek** [microsoft/vscode-remote-release#6113]
Issue Type: <b>Bug</b>

I am trying to troubleshoot a ssh disconnect issue with Remote-SSH. To help do that I am trying to increase logging from ssh, but it doesn't seem to be working (though it works with regular command line).

1) Set LogLevel DEBUG2 in .ssh/config.
2) Connect to remote host in vscode

Expected: See debug1/debug2 entries in Remote-SSH log
Actual: No debug1/debug2 entries from ssh

Extension version: 0.70.0
VS Code version: Code 1.63.2 (Universal) (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:37:28.172Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No
Fetching remote diagnostics for 'SSH: drees-m920-vscode' failed: Canceled
Remote OS version: Linux x64 5.10.46-5rodete1-amd64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|8, 8, 8|
|Memory (System)|32.00GB (0.35GB free)|
|Process Argv|--crash-reporter-id 007368ee-5839-4f03-a024-ccd293d8d010|
|Screen Reader|no|
|VM|0%|

Fetching remote diagnostics for 'SSH: drees-m920-vscode' failed: Canceled

|Item|Value|
|---|---|
|Remote|SSH: drees-m920-vscode|
|OS|Linux x64 5.10.46-5rodete1-amd64|
|CPUs|Intel(R) Xeon(R) Gold 6154 CPU @ 3.00GHz (72 x 1200)|
|Memory (System)|187.52GB (22.42GB free)|
|VM|0%|
</details>

**Comment by studgeek** [comment]
Here is the .ssh/config stanza:

```
Host drees-m920-vscode 
  HostName drees-m920.XXX
  LogLevel DEBUG2
```

Here is Remote-SSH log:
```
[09:09:38.771] Log Level: 2
[09:09:38.773] remote-ssh@0.70.0
[09:09:38.773] darwin x64
[09:09:38.774] SSH Resolver called for "ssh-remote+drees-m920-vscode", attempt 1
[09:09:38.774] "remote.SSH.useLocalServer": true
[09:09:38.774] "remote.SSH.path": undefined
[09:09:38.774] "remote.SSH.configFile": undefined
[09:09:38.775] "remote.SSH.useFlock": true
[09:09:38.775] "remote.SSH.lockfilesInTmp": false
[09:09:38.775] "remote.SSH.localServerDownload": auto
[09:09:38.775] "remote.SSH.remoteServerListenOnSocket": false
[09:09:38.775] "remote.SSH.showLoginTerminal": false
[09:09:38.775] "remote.SSH.defaultExtensions": []
[09:09:38.775] "remote.SSH.loglevel": 2
[09:09:38.776] "remote.SSH.serverPickPortsFromRange": {}
[09:09:38.776] "remote.SSH.enableDynamicForwarding": true
[09:09:38.776] "remote.SSH.serverInstallPath": {}
[09:09:38.777] SSH Resolver called for host: drees-m920-vscode
[09:09:38.777] Setting up SSH remote "drees-m920-vscode"
[09:09:38.779] Acquiring local install lock: /var/folders/ms/6405ktc536ldff05tgtfqzxw0062hb/T/vscode-remote-ssh-2d82d29e-install.lock
[09:09:38.786] Looking for existing server data file at /Users/drees/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-2d82d29e-899d46d82c4c95423fb7e10e68eba52050e30ba3-0.70.0/data.json
[09:09:38.786] Using commit id "899d46d82c4c95423fb7e10e68eba52050e30ba3" and quality "stable" for server
[09:09:38.789] Install and start server if needed
[09:09:38.795] PATH:...
[09:09:38.795] Checking ssh with "ssh -V"
[09:09:38.805] > OpenSSH_8.8p1, OpenSSL 1.1.1l  24 Aug 2021

[09:09:38.812] askpass server listening on /var/folders/ms/6405ktc536ldff05tgtfqzxw0062hb/T/vscode-ssh-askpass-3607a15cab6173b9cff88d432e842e58dc8e8979.sock
[09:09:38.813] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/ms/6405ktc536ldff05tgtfqzxw0062hb/T/vscode-ssh-askpass-e995709078f92396255ac58ef2f902e43a3a22b6.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","56672","-o","ConnectTimeout=60","drees-m920-vscode","bash"],"dataFilePath":"/Users/drees/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-2d82d29e-899d46d82c4c95423fb7e10e68eba52050e30ba3-0.70.0/data.json"}
[09:09:38.813] Local server env: {"DISPLAY":"/private/tmp/com.apple.launchd.gdKxlIrB8K/org.xquartz:0","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/drees/.vscode/extensions/ms-vscode-remote.remote-ssh-0.70.0/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/MacOS/Electron","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"--ms-enable-electron-run-as-node","VSCODE_SSH_ASKPASS_MAIN":"/Users/drees/.vscode/extensions/ms-vscode-remote.remote-ssh-0.70.0/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/ms/6405ktc536ldff05tgtfqzxw0062hb/T/vscode-ssh-askpass-3607a15cab6173b9cff88d432e842e58dc8e8979.sock"}
[09:09:38.814] Spawned 88279
[09:09:38.929] > local-server-1> Spawned ssh, pid=88306
[09:09:38.933] stderr> OpenSSH_8.8p1, OpenSSL 1.1.1l  24 Aug 2021
[09:09:45.230] stderr> debug1: Server host key: ssh-rsa SHA256:Z4fZ5bnoHrpZuNwH8V4uocpVmSWR/SOwsIDOyJLjU/s
[09:09:50.130] stderr> Authenticated to drees-m920.XXX (via proxy) using "publickey".
[09:09:50.316] > ready: 9bccb2b1adac
[09:09:50.340] > Linux 5.10.46-5rodete1-amd64 #1 SMP Debian 5.10.46-5rodete1 (2021-09-28)
[09:09:50.352] Platform: linux
[09:09:50.392] > 9bccb2b1adac: running
[09:09:50.436] > Acquiring lock on /usr/local/google/home/drees/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/vscode-remote-lock.drees.899d46d82c4c95423fb7e10e68eba52050e30ba3
[09:09:50.446] > Found existing installation at /usr/local/google/home/drees/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3...
> Checking /usr/local/google/home/drees/.vscode-server/.899d46d82c4c95423fb7e10e68eba52050e30ba3.log and /usr/local/google/home/drees/.vscode-server/.899d46d82c4c95423fb7e10e68eba52050e30ba3.pid for a running server
> Looking for server with pid: 3202748
[09:09:50.504] > Found running server...
[09:09:50.508] >  
> *
> * Reminder: You may only use this software with Visual Studio family products,
> * as described in the license (https://go.microsoft.com/fwlink/?linkid=2077057)
> *
>  
[09:09:50.525] > Checking server status on port 40493 with wget
[09:09:50.539] > 9bccb2b1adac: start
> SSH_AUTH_SOCK==/tmp/ssh-XXXXJsasuu/agent.3364230==
> DISPLAY====
> webUiAccessToken====
> listeningOn==40493==
> osReleaseId==debian==
> arch==x86_64==
> tmpDir==/run/user/199179==
> platform==linux==
> unpackResult====
> didLocalDownload==0==
> downloadTime====
> installTime====
> extInstallTime====
> serverStartTime====
> connectionToken==a11a1111-aa11-11a1-aa1a-a1111a111111==
> 9bccb2b1adac: end
[09:09:50.539] Received install output: 
SSH_AUTH_SOCK==/tmp/ssh-XXXXJsasuu/agent.3364230==
DISPLAY====
webUiAccessToken====
listeningOn==40493==
osReleaseId==debian==
arch==x86_64==
tmpDir==/run/user/199179==
platform==linux==
unpackResult====
didLocalDownload==0==
downloadTime====
installTime====
extInstallTime====
serverStartTime====
connectionToken==a11a1111-aa11-11a1-aa1a-a1111a111111==

[09:09:50.540] Remote server is listening on 40493
[09:09:50.540] Parsed server configuration: {"serverConfiguration":{"remoteListeningOn":{"port":40493},"osReleaseId":"debian","arch":"x86_64","webUiAccessToken":"","sshAuthSock":"/tmp/ssh-XXXXJsasuu/agent.3364230","display":"","tmpDir":"/run/user/199179","platform":"linux","connectionToken":"a11a1111-aa11-11a1-aa1a-a1111a111111"},"installUnpackCode":""}
[09:09:50.541] Persisting server connection details to /Users/drees/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-2d82d29e-899d46d82c4c95423fb7e10e68eba52050e30ba3-0.70.0/data.json
[09:09:50.545] Starting forwarding server. localPort 56690 -> socksPort 56672 -> remotePort 40493
[09:09:50.545] Forwarding server listening on 56690
[09:09:50.546] Waiting for ssh tunnel to be ready
[09:09:50.547] Tunneled 40493 to local port 56690
[09:09:50.547] Resolved "ssh-remote+drees-m920-vscode" to "127.0.0.1:56690"
[09:09:50.547] Updating $SSH_AUTH_SOCK: ln -f -s "/tmp/ssh-XXXXJsasuu/agent.3364230" "/run/user/199179/vscode-ssh-auth-sock-782715095"
[09:09:50.547] Using cwd: vscode-remote://ssh-remote%2Bdrees-m920-vscode/
[09:09:50.548] Remote extension host environment: {"SSH_AUTH_SOCK":"/run/user/199179/vscode-ssh-auth-sock-782715095"}
[09:09:50.549] [Forwarding server 56690] Got connection 0
[09:09:50.555] ------




[09:09:50.564] [Forwarding server 56690] Got connection 1
[09:09:50.612] [Forwarding server 56690] Got connection 2
[09:09:52.301] [Forwarding server 56690] Got connection 3
[09:09:52.319] [Forwarding server 56690] Got connection 4
[09:09:52.998] "Update SSH_AUTH_SOCK" terminal command done
```

**Comment by bamurtaugh** [comment]
Thanks for filing @studgeek. I see `stderr> debug1: Server host key: ssh-rsa SHA256:Z4fZ5bnoHrpZuNwH8V4uocpVmSWR/SOwsIDOyJLjU/s` in the output provided - but you were expecting to see `debug1` and `debug2` entries?
